docs/bundles/plain.md

```diff
@@ -29,16 +60,125 @@ manifests
+  location in order to have a valid bundle git repo. If a specific directory is not provided, it is assumed to be
+  ./manifests in a bundle git repo.
 * The manifests directory should be flat: all manifests should be at the top-level with no subdirectories.
-* Including any content in the /manifests directory of a plain bundle that is not static manifests will result in
-a failure when creating content on-cluster from that bundle. Essentially, any file that would not
-successfully `kubectl apply` will result in an error, but multi-object YAML files, or JSON files, are fine. There will
-be validation tooling provided that can determine whether a given artifact is a valid bundle.
\ No newline at end of file
+* The plain bundle image can be built from any base image, but `scratch` is recommended as it keeps the resulting bundle
+  image a minimal size.
+* Including any content in the root `manifests` directory of a plain bundle that is not static manifests will result in
+  a failure when creating content on-cluster from that bundle via
+  a [BundleDeployment](https://github.com/operator-framework/rukpak#bundledeployment). Essentially, any file that would not
+  successfully `kubectl apply` will result in an error, but multi-object YAML files, or JSON files, are fine. There will
+  be validation tooling provided that can determine whether a given artifact is a valid bundle.
+
+## Building a plain bundle
+### Prerequisites
+
+In order to create a plain bundle for Rukpak, ensure your Kubernetes manifests are in a flat directory at the root of your project called `manifests/`. This allows the contents to be sourced and unpacked by the [plain provisioner](../provisioners/plain.md). It should look similar to:
+
+```bash
+$ tree manifests
+manifests
+├── namespace.yaml
+├── service_account.yaml
+├── cluster_role.yaml
+├── cluster_role_binding.yaml
+└── deployment.yaml
+```
+
+If you are using [kustomize](https://kustomize.io/) for building your manifests from a template, redirect the output into a single file under the `manifests/` directory. For example:
+```bash
+$ tree templates
+templates
+├── namespace.yaml
+├── service_account.yaml
+├── cluster_role.yaml
+├── cluster_role_binding.yaml
+├── deployment.yaml.yaml
+└── kustomization.yaml
+```
+
+```bash
+kustomize build templates > manifests/manifests.yaml
+```
+
+### Git source
+
+For using the bundle with a [git source](../sources/git.md), commit your `manifests/` directory to your git repository.
+
+### HTTP source
+
+For using the bundle with a [HTTP source](../sources/http.md), create a `.tgz` file that contains the `manifests/` directory and it's manifests. This `.tgz` file should be served when hitting the HTTP endpoint.
+
+### Image source
+
+For using the bundle with an [image source](../sources/image.md), follow the below steps:
+
+1. Create a Dockerfile at the root of the project
+```bash
+touch Dockerfile.plainbundle
```

Comment:
```suggestion
$ touch Dockerfile.plainbundle
```

Reply:
addressed in https://github.com/operator-framework/rukpak/pull/642/commits/486faf8915e2f3f891ee769b99651e3039d052b1